src/igvfd/schemas/human_donor.json

```diff
@@ -44,12 +44,51 @@
     ],
     "properties": {
         "schema_version": {
-            "default": "7"
+            "default": "8"
         },
-        "parents": {
+        "related_donors": {
+            "title": "Related Donors",
+            "description": "Familial relations of this donor",
+            "type": "array",
+            "minItems": 1,
+            "uniqueItems": true,
             "items": {
-                "linkTo": "HumanDonor",
-                "type": "string"
+                "title": "Related Donors",
+                "description": "Familial relations of this donor",
+                "type": "object",
+                "additionalProperties": false,
+                "required": [
+                    "donor",
+                    "relationship_type"
+                ],
+                "properties": {
+                    "donor": {
+                        "title": "Related Donor Identifier",
+                        "description": "An identifier for the related donor.",
+                        "type": "string",
+                        "linkTo": "HumanDonor"
+                    },
+                    "relationship_type": {
+                        "title": "Relationship Type",
+                        "description": "A descriptive term for the related donor’s relationship to this donor.",
+                        "type": "string",
+                        "enum": [
```

Comment:
can you please add comment with URL or explanation where the relationships list are coming from?

Reply:
ok